src/pages/about-us/components/_hero.tsx

```diff
@@ -100,23 +99,23 @@ const Hero = () => {
                             alt="example"
                             width="100%"
                         />
-                        <FlexHeader>
-                            <StyledHeaderDesktopAbout
-                                as="h1"
-                                color="white"
-                                align="center"
-                                type="unset"
-                            >
-                                {localize('About us')}
-                            </StyledHeaderDesktopAbout>
-                        </FlexHeader>
+                        <StyledHeaderDesktopAbout
+                            as="h1"
+                            color="white"
+                            align="center"
+                            type="main-landing-title"
+                            width="100%"
+                            padding="0"
+                        >
+                            {localize('Who we are')}
+                        </StyledHeaderDesktopAbout>
                     </StyledFlex>
                 </DesktopWrapper>
                 <MobileWrapper>
                     <Flex fd="column">
                         <QueryImage data={data['about_us_logo']} alt="example" />
                         <StyledHeader as="h1" color="white" align="center" mt="40px" type="unset">
-                            {localize('About us')}
+                            {localize('Who \nwe are')}
```

Comment:
Please either do it in one of these 2 ways as it would be much easier for the translators since they wouldn't know what `\n` would mean.

```
{localize('Who')}
<br></br>
{localize('we are')}
```
or
```
<Localize
    translate_text="Who<0 />we are"
    components={[<br key={0} />]}
/>
```

Reply:
done